Source/Microsoft.Teams.Apps.CompanyCommunicator.Common/Repositories/NotificationData/NotificationDataEntity.cs

```diff
@@ -156,5 +156,20 @@ public IEnumerable<string> Rosters
         /// Gets or sets the sending started DateTime value.
         /// </summary>
         public DateTime? SendingStartedDate { get; set; }
+
+        /// <summary>
+        /// Gets or sets the exception message.
```

Comment:
@jotrick Can you elaborate on these? What's an exception message vs a warning message?

Reply:
Will look into/change after completion of this PR.